src/utils.js

```diff
@@ -497,3 +497,17 @@ export const md5Hash = item => {
   md5sum.update(JSON.stringify(item));
   return md5sum.digest('hex');
 };
+
+export const isEncodingType = encodingType =>
+  !!(
+    encodingType &&
+    typeof encodingType === 'string' &&
+    ['ascii', 'utf8', 'utf16le', 'ucs2', 'base64', 'binary', 'hex'].includes(
+      encodingType
+    )
+  );
+
+export const isEmptyString = str => !!(!str || str.length === 0);
```

Comment:
add `typeof` to avoid `isEmptyString([])`

Reply:
👍